docs/reference/limitations.rst

```diff
@@ -0,0 +1,50 @@
+.. _reference-limitations:
+
+***********
+Limitations
+***********
+
```

Comment:
The issue with the `license` key in `pyproject.toml` is probably worth pointing out.